README.md

```diff
@@ -106,15 +106,17 @@ Commands:
   completion  generate bash completion script
 
 Options:
-  --config, -c     Config file to use (defaults to nearest package-scripts.yml
-                   or package-scripts.js)
-                     [default: "<path-to-your-project>/package-scripts.js"]
-  --silent, -s     Silent nps output                  [boolean] [default: false]
-  --log-level, -l  The log level to use
-                   [choices: "error", "warn", "info", "debug"] [default: "info"]
-  --require, -r    Module to preload
-  -h, --help       Show help                                           [boolean]
-  -v, --version    Show version number                                 [boolean]
+  --config, -c      Config file to use (defaults to nearest package-scripts.yml
+                    or package-scripts.js)
+                      [default: "<path-to-your-project>/package-scripts.js"]
+  --silent, -s      Silent nps output                  [boolean] [default: false]
+  --log-level, -l   The log level to use
+                    [choices: "error", "warn", "info", "debug"] [default: "info"]
+  --require, -r     Module to preload
+  -h, --help        Show help                                           [boolean]
+  -v, --version     Show version number                                 [boolean]
+  --help-style, -hs Style of help to use
```

Comment:
I think that `-hs` should be `-y` based on the alias in the parser.

Reply:
Yup, good catch.